main.go

```diff
@@ -65,13 +75,20 @@ func main() {
 	pflag.StringVar(&hubConfigSecretNs, "hubconfig-secret-ns", "open-cluster-management-agent-addon", "Namespace for hub config kube-secret")
 	pflag.StringVar(&hubConfigSecretName, "hubconfig-secret-name", "policy-controller-hub-kubeconfig", "Name of the hub config kube-secret")
 
+	var enableLeaderElection bool
+	var probeAddr string
+	pflag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	pflag.BoolVar(&enableLeaderElection, "leader-elect", false,
```

Comment:
I would set `leader-elect` to `true` and then remove it from the container arguments in the `Deployment` to avoid a change to the Helm charts.

I think in general, it's better to have leader election on by default since this controller wouldn't play well with another one running at the same time both acting as leaders.

Reply:
I agree